sample/src/main/java/com/uber/rave/sample/DefaultSerializer.java

```diff
@@ -0,0 +1,53 @@
+package com.uber.rave.sample;
+
+import com.uber.rave.sample.github.model.Owner;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+
+/**
+ * This uses the default Java serializer.
+ */
+class DefaultSerializer {
+
+    static byte[] serialize(Owner owner) {
```

Comment:
SharedPrefs doesn't have a byte array write. There's also nothing to gain from actually writing to disk vs just simulating the write. The write part doesn't have anything to do with RAVE, it's only the serialization part.